File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/publiseringsdatoer/api/Publiseringsdatoer.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.publiseringsdatoer.api;

import lombok.Builder;
import lombok.Data;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles.ÅrstallOgKvartal;

@Data
@Builder
public class Publiseringsdatoer {
    private String sistePubliseringsdato;
    private String nestePubliseringsdato;
    private ÅrstallOgKvartal gjeldendePeriode;
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/publiseringsdatoApi/Publiseringsdatoer.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.publiseringsdatoApi

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.fellesdomene.ÅrstallOgKvartal
import java.time.LocalDate

data class Publiseringsdatoer(
    val sistePubliseringsdato: LocalDate,
    val gjeldendePeriode: ÅrstallOgKvartal,
    val nestePubliseringsdato: LocalDate?,
)
